forte/solvers/hf.py

```diff
@@ -12,30 +11,31 @@ class HF(Solver):
     """
     def __init__(
         self,
-        parent_solver,
+        input,
         state,
         restricted=True,
         e_convergence=1.0e-10,
         d_convergence=1.0e-6,
-        int_type='conventional',
         docc=None,
         socc=None,
         options=None,
         cbh=None
     ):
         """
-        initialize a Basis object
+        initialize a HF object
 
         Parameters
         ----------
-        restricted : bool
+        input: Solver
```

Comment:
Why rename this variable to input? It's vague.

Reply:
This is probably transitory. I am trying to stick to the terminology of nodes where you have input and output nodes. Maybe use `input_solver`? `parent_solver` is also reasonable and I could be swayed to go back to it.